src/Sylius/Component/Core/OrderPaymentTransitions.php

```diff
@@ -18,6 +18,8 @@
     public const GRAPH = 'sylius_order_payment';
 
     public const TRANSITION_REQUEST_PAYMENT = 'request_payment';
+    public const TRANSITION_PARTIALLY_AUTHORIZE_PAYMENT = 'partially_authorize_payment';
+    public const TRANSITION_AUTHORIZE_PAYMENT = 'authorize_payment';
```

Comment:
I'm wondering about those transitions: Should they have `payment` suffix?

Reply:
Good point, I would vote for removal

Reply:
So do I, it adds unneeded characters that can degrade visiblity of code.

Reply:
Fixed!